packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/CompleteProfile/LinkItem/index.tsx

```diff
@@ -58,6 +60,7 @@ const LinkItem = ({ isComplete, onClick, type }: Props) => {
 
 type Props = {
   isComplete: boolean
+  isPending?: boolean
```

Comment:
this is pretty confusing as `isComplete` vs `isPending` mean the same thing to me as they are named right now. can we name the props better?

Reply:
yeah we can like isProcessing and isComplete, wdyt?